codersdk/deployment.go

```diff
@@ -530,3 +545,35 @@ func (c *Client) AppHost(ctx context.Context) (AppHostResponse, error) {
 	var host AppHostResponse
 	return host, json.NewDecoder(res.Body).Decode(&host)
 }
+
+type WorkspaceConnectionLatencyMS struct {
+	P50 float64
+	P95 float64
+}
+
+type DeploymentStats struct {
+	// AggregatedFrom is the time in which stats are aggregated from.
+	// This might be back in time a specific duration or interval.
+	AggregatedFrom time.Time `json:"aggregated_since" format:"date-time"`
+	// CollectedAt is the time in which stats are collected at.
+	CollectedAt time.Time `json:"collected_at" format:"date-time"`
+	// RefreshingAt is the time when the next batch of stats will
+	// be refreshed.
+	RefreshingAt time.Time `json:"refreshing_at" format:"date-time"`
```

Comment:
Suggestion (optional): This could be named `NextUpdateAt`, basically refresh vs update, I feel the latter is a bit easier to understand.

Reply:
Agreed